.travis.yml

```diff
@@ -72,8 +72,11 @@ before_script:
 script:
     - make
 
-    # Run all examples
-    - make run-examples
+    # Run tests with catch
```

Comment:
nit, I think you can either use one comment for both sets of tests, or make the comments more specific ("Run mongocxx tests")